GitUI/UserControls/BlameControl.cs

```diff
@@ -307,12 +317,15 @@ private void ProcessBlame(string filename, GitRevision revision, IReadOnlyList<O
             controlToMask?.UnMask();
         }
 
-        private (string gutter, string body) BuildBlameContents(string filename)
+        private (string gutter, string body, List<GitBlameEntry> gitBlameDisplays) BuildBlameContents(string filename,
+            int avatarSize)
         {
             var body = new StringBuilder(capacity: 4096);
 
             GitBlameCommit lastCommit = null;
 
+            var gitBlameDisplays = AppSettings.BlameShowAuthorAvatar ? CalculateBlameGutterData(_blame.Lines) : null;
```

Comment:
It is a bad practice to return `null` for enumerable types. This can easily lead to NREs, as it is very unexpected.